tests/lib/nested.sh

```diff
@@ -52,7 +52,19 @@ nested_wait_for_no_ssh() {
 }
 
 nested_wait_for_snap_command() {
-    nested_exec "retry --wait 1 -n 200 sh -c 'command -v snap'"
+    # In this function cannot be used the remote retry command because it could
```

Comment:
nitpick: "In this function the remote retry command cannot be used because..."

Reply:
Do I understand correctly, that the problem is that the `retry` command itself might not be there at this time?

Reply:
Right, when we use nested_wait_for_snap_command after creating/booting a new image, the retry command is not yet deployed.

Reply:
Comment updated, thanks for review